instrumentation/trilogy/lib/opentelemetry/instrumentation/trilogy/instrumentation.rb

```diff
@@ -61,10 +45,10 @@ def configure_propagator(config)
           propagator = config[:propagator]
           @propagator = case propagator
                         when 'vitess' then fetch_propagator(propagator, 'OpenTelemetry::Propagator::Vitess')
-                        when 'none', nil then NoopPropagator.new
+                        when 'none', nil
+                          nil
```

Comment:
```suggestion
                        when 'none', nil then nil
```
for consistency (otherwise remove the `then` in the `when` above).